extractor.go

```diff
@@ -17,6 +17,9 @@ const bearer = "bearer"
 // UserKey ...
 const UserKey = "user"
 
+// Algorithm used for signing
+var Alg = jwt.SigningMethodHS256.Alg()
+
 // TokenExtractor Attempts to extract the token on every request
 func TokenExtractor(logger log.Logger, secret interface{}) kithttp.RequestFunc {
```

Comment:
We can always make it a pointer, and if it's nil just use the default. The reason suggested it is that this repo is public, meaning someone may want to use this utility for their own purposes.